packages/skeleton-svelte/src/lib/components/Switch/Switch.svelte

```diff
@@ -54,9 +56,7 @@
 		zagSwitch.machine({
 			id: useId(),
 			name,
-			onCheckedChange(details) {
-				checked = details.checked;
```

Comment:
Removing this line means the `bind:checked` no longer will work as it's never updated. Please revert.